l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py

```diff
@@ -173,6 +173,30 @@ def test_05_xml_import(self):
         self.assertEqual(invoice.amount_tax, 0)
         self.assertEqual(invoice.amount_total, 9)
 
+    def test_xml_import_with_action_cancel(self):
+        res = self.run_wizard(
+            "test5.2",
+            "IT05979361218_003_action_cancel.xml",
+            wizard_check_action="action_cancel",
+        )
+        invoice_id = res.get("domain")[0][2][0]
+        invoice = self.invoice_model.browse(invoice_id)
+        self.assertEqual(invoice.ref, "FT/2015/0030")
+        self.assertEqual(invoice.payment_reference, "FT/2015/0030")
+        self.assertEqual(invoice.sender, "TZ")
+        self.assertNotEqual(invoice.intermediary.name, "LUIGI BIANCHI")
+        self.assertNotEqual(invoice.intermediary.firstname, "LUIGI")
+        self.assertNotEqual(invoice.intermediary.lastname, "BIANCHI")
```

Comment:
Basta verificare che non c'è il partner
```suggestion
        self.assertFalse(invoice.intermediary)
```

Reply:
Grazie, tutti gli altri `assert` si potrebbero eliminare perché l'unica cosa che deve verificare questo test è che non venga creato il terzo intermediario.

Potresti aggiungere un `assert` per verificare che il partner non sia stato creato?
Magari potrebbe essere stato creato ma non collegato alla fattura.